rest/route/project.go

```diff
@@ -962,6 +923,147 @@ func (h *getProjectVersionsHandler) Run(ctx context.Context) gimlet.Responder {
 	return resp
 }
 
+// POST /rest/v2/projects/{project_id}/versions
+
+// versionsSetPriorityHandler is a RequestHandler for setting the priority of versions.
```

Comment:
I know that we currently only let users bump the priority on patches up to a certain number, unless they have special permissions to bypass it. Do we have that check in place here? I know we are creating this route for disabling, but it's also giving users the power to bump the priority up on a bunch of versions, potentially blocking important versions from running for a while.

Reply:
Yep, the route hits [this function](https://gist.github.com/hadjri/0a8e4e99a2df70aee805591d2cd1a27b#file-middleware-go-L116) which will allow those with `TasksAdmin` role to surpass the max priority